src/azure-cli/azure/cli/command_modules/synapse/_params.py

```diff
@@ -580,3 +580,86 @@ def load_arguments(self, _):
     with self.argument_context('synapse notebook delete') as c:
         c.argument('workspace_name', arg_type=workspace_name_arg_type)
         c.argument('notebook_name', arg_type=name_type, help='The notebook name.')
+
+    # synapse integration runtime
+    with self.argument_context('synapse integration-runtime show') as c:
+        c.argument('workspace_name', arg_type=workspace_name_arg_type, id_part='name')
+        c.argument('integration_runtime_name', arg_type=name_type, help='The integration runtime name.', id_part='child_name_1')
+
+    with self.argument_context('synapse integration-runtime list') as c:
+        c.argument('workspace_name', arg_type=workspace_name_arg_type)
+
+    with self.argument_context('synapse integration-runtime create') as c:
+        c.argument('workspace_name', arg_type=workspace_name_arg_type, id_part='name')
+        c.argument('integration_runtime_name', arg_type=name_type, help='The integration runtime name.', id_part='child_name_1')
+        c.argument('integration_runtime_type', options_list=['--type'], arg_type=get_enum_type(['Managed', 'SelfHosted']), help='The integration runtime type.')
+        c.argument('description', help='The integration runtime description.')
+        c.argument('if_match', help='ETag of the integration runtime entity. Should only be specified for update, for '
+                   'which it should match existing entity or can be * for unconditional update.')
+        # Managed
+        c.argument('location', arg_group='Managed', help='The integration runtime location.')
+        c.argument('compute_type', arg_group='Managed', arg_type=get_enum_type(['General', 'MemoryOptimized', 'ComputeOptimized']),
+                   help='Compute type of the data flow cluster which will execute data flow job.')
+        c.argument('core_count', arg_group='Managed', help='Core count of the data flow cluster which will execute data flow job.')
+        c.argument('time_to_live', arg_group='Managed', help='Time to live (in minutes) setting of the data flow cluster which will execute data flow job.')
+
+    with self.argument_context('synapse integration-runtime delete') as c:
+        c.argument('workspace_name', arg_type=workspace_name_arg_type, id_part='name')
+        c.argument('integration_runtime_name', arg_type=name_type, help='The integration runtime name.', id_part='child_name_1')
+
+    with self.argument_context('synapse integration-runtime update') as c:
+        c.argument('workspace_name', arg_type=workspace_name_arg_type, id_part='name')
+        c.argument('integration_runtime_name', arg_type=name_type, help='The integration runtime name.', id_part='child_name_1')
```

Comment:
Define abbreviation such as `--runtime` for easy use for customers; consider other long arguments as well.

Reply:
I just use the argument `--name` or `-n` here.